### ISSUE TITLE
Create Base Docker Images

### ISSUE THREAD

**Issue by kmader** [DeepInfer/Model-Registry#3]
Quite a bit of the Docker file is strongly predefined (paths, users, sitk dependencies, ...). It would be good to have base docker files covering python 2.7/3.5, cpu/gpu, and ideally theano, tensorflow, torch, pytorch, caffe

**Comment by mpslxz** [comment]
@kmader You are absolutely right. I was thinking about making a base image with the updated libraries installed, which I'll probably do. But the only thing is some people do use other versions of different libraries so I am not sure how useful and practical it would be if we have a "base image".
